ethdk/.eslintrc.js

```diff
@@ -16,5 +16,6 @@ module.exports = {
   rules: {
     indent: ['error', 2],
     'prettier/prettier': 'error',
+    '@typescript-eslint/no-unnecessary-boolean-literal-compare': 'off',
```

Comment:
Disabling this rule allows us to do this:
```ts
    if (validatedPrivateKey.success === false) {
      throw new Error(`Invalid private key: ${validatedPrivateKey.error}`)
    }
```

Without this rule, we get an error and the fix is this:
```ts
    if (!validatedPrivateKey.success) {
      throw new Error(`Invalid private key: ${validatedPrivateKey.error}`)
    }
```

I think the first option is more readable. This is fairly opinionated though so feel free to disagree with this change :)

Reply:
I think that's a fair statement to say the first one is more readable.  I'm cool with disabling this rule 👍